libraries/SD_MMC/src/SD_MMC.h

```diff
@@ -16,7 +16,9 @@
 
 #include "sdkconfig.h"
 #include "soc/soc_caps.h"
-#ifdef SOC_SDMMC_HOST_SUPPORTED
+#ifndef SOC_SDMMC_HOST_SUPPORTED
+#error The SDMMC library requires a device with an SDIO Host
+#else
```

Comment:
the way it was, was the better way. IDF compiles everything, regardless of the target, so this will break compilation for chips other than S3

Reply:
@lucasssvaz I suggest the IDF CI runs any time lib or core is changed (not examples)

Reply:
@me-no-dev Ok, i'm currently in the process of fixing some issues I found with the CI. I can include this.

Reply:
Is there a way to have IDF compile cleanly yet have a arduino compile time warning? This ends up giving a somewhat confusing error that the functions are not available if a newbie decides to try this on an unsupported platform.

Reply:
it can be a warning. A bit annoying, but will work. You can probably detect if it's Arduino by some define too

Reply:
Better?

Reply:
better :)